modules/netstat/win_test.go

```diff
@@ -0,0 +1,194 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Contributor: Rob Murtha robmurtha@gmail.com [:robmurtha]
+package netstat
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
```

Comment:
I added the testify package to reduce overhead when writing tests, it's one of the smaller packages